protocol/get_ledgers.go

```diff
@@ -74,3 +57,51 @@ type GetLedgersResponse struct {
 	OldestLedgerCloseTime int64        `json:"oldestLedgerCloseTime"`
 	Cursor                string       `json:"cursor"`
 }
+
+// IsStartLedgerWithinBounds checks whether the request start ledger/cursor is
+// within the max/min ledger for the current RPC instance.
+func IsStartLedgerWithinBounds(startLedger uint32, ledgerRange LedgerSeqRange) bool {
+	return startLedger >= ledgerRange.FirstLedger && startLedger <= ledgerRange.LastLedger
```

Comment:
Not required, but I think this function in its name and description describe more of a use case it is used for, rather than the simple primitive it provides, and it could be reduced to:
```suggestion
// IsLedgerWithinBounds checks whether the request ledger is within the range specified.
func IsLedgerWithinRange(ledger uint32, ledgerRange LedgerSeqRange) bool {
	return ledger >= ledgerRange.FirstLedger && ledger <= ledgerRange.LastLedger
```

Reply:
Yeah good call, done